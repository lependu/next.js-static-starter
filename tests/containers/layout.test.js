/* eslint-env jest */
import renderer from 'react-test-renderer'
import Layout from 'containers/layout'

describe('Container | Layout', () => {
  test('matches snapshot', () => {
    let wrapper = renderer.create(<Layout id='index' />)
    let tree = wrapper.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
