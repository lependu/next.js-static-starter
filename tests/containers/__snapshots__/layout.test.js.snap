// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container | Layout matches snapshot 1`] = `
<div
  id="site-layout"
>
  <nav
    id="site-navigation"
  >
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          index
        </a>
      </li>
      <li>
        <a
          href="/other"
          onClick={[Function]}
        >
          other
        </a>
      </li>
    </ul>
  </nav>
  <div>
    <h1>
      Index page
    </h1>
    <p>
       
    </p>
    <hr />
    <p>
       
    </p>
    <p>
      This is the index page content.
    </p>
  </div>
</div>
`;
